### ISSUE TITLE
NoiseScheduler parameters for StableDiffusion are wrong

### ISSUE THREAD

**Issue by DaniyarM** [keras-team/keras-cv#2049]
NoiseScheduler [https://github.com/keras-team/keras-cv/blob/master/keras_cv/models/stable_diffusion/noise_scheduler.py](url) contains a bug with default parameters. For a pretrained SD it must be: NoiseScheduler(beta_start=0.00085, beta_end=0.012, beta_schedule='scaled_linear', clip_sample=False). For now it is: NoiseScheduler(beta_start=0.0001, beta_end=0.02, beta_schedule='linear', clip_sample=True). As I understand it, this happened because of copy-paste with a hugging face (where the default parameters are taken for some old GANs).

**Comment by jbischof** [comment]
Thanks @DaniyarM! Can you share the source of the corrected parameters?

**Comment by DaniyarM** [comment]
It can be obtained from huggingface's diffusers like that :
`from diffusers import DDIMScheduler`
`scheduler = DDIMScheduler.from_pretrained('runwayml/stable-diffusion-v1-5', subfolder='scheduler')`
`print(scheduler)`

Also, it can be found in that article: [https://arxiv.org/pdf/2305.08891v2.pdf](url) (Table 1).

**Comment by ianstenbit** [comment]
Thanks @DaniyarM for pointing this out -- feel free to send a PR to update this!

**Comment by sachinprasadhs** [comment]
Thanks for reporting the issue! We have consolidated the development of KerasCV into the new **KerasHub** package, which supports image, text, and multi-modal models. Please read [the announcement](https://github.com/keras-team/keras-hub/issues/1831). `KerasHub` will support all the core functionality of `KerasCV`.

`KerasHub` can be installed with `!pip install -U keras-hub`. Documentation and guides are available at [keras.io/keras_hub](https://keras.io/keras_hub/).

With our focus shifted to KerasHub, we are not planning any further development or releases in KerasCV. If you encounter a KerasCV feature that is missing from KerasHub, or would like to propose an addition to the library, please file [an issue with KerasHub](https://github.com/keras-team/keras-hub/issues).
